condaci.py

```diff
@@ -312,32 +319,22 @@ def get_conda_build_path(recipe_dir):
         raise ValueError('Unable to find recipe_dir')
 
 
-def conda_build_package_win(mc, path):
-    if 'BINSTAR_KEY' in os.environ:
-        print('found BINSTAR_KEY in environment on Windows - deleting to '
-              'stop vcvarsall from telling the world')
-        del os.environ['BINSTAR_KEY']
-    os.environ['PYTHON_ARCH'] = host_arch()[:2]
-    os.environ['PYTHON_VERSION'] = PYTHON_VERSION
-    print('PYTHON_ARCH={} PYTHON_VERSION={}'.format(os.environ['PYTHON_ARCH'],
-                                                    os.environ['PYTHON_VERSION']))
-    execute([conda(mc), 'build', '-q', path,
-             '--py={}'.format(PYTHON_VERSION_NO_DOT)])
-
-
 def windows_setup_compiler():
-    arch = host_arch()
-    if PYTHON_VERSION in VS9_PY_VERS and arch == '64bit':
+    if PYTHON_VERSION in VS9_PY_VERS and ARCH == 'x64':
         VS2008_AMD64_PATH = os.path.join(VS2008_BIN_PATH, 'amd64')
         if not os.path.exists(VS2008_AMD64_PATH):
             os.makedirs(VS2008_AMD64_PATH)
         VCVARS64_PATH = os.path.join(VS2008_BIN_PATH, 'vcvars64.bat')
         VCVARSAMD64_PATH = os.path.join(VS2008_AMD64_PATH, 'vcvarsamd64.bat')
-        print("Copying '{}' to '{}' to fix VS2008 64-bit configuration.".format(
-                  VCVARS64_PATH, VCVARSAMD64_PATH))
-        shutil.copyfile(VCVARS64_PATH, VCVARSAMD64_PATH)
+        if not os.path.exists(VCVARS64_PATH):
+            print("Unable to find '{}' - skipping fix for VS2008 64-bit.".format(
+                VCVARS64_PATH))
```

Comment:
Can it be OK to skip this? or should we be raising an error if Py 2.7/64bit? (I'm guessing)